lib/internal/bootstrap_node.js

```diff
@@ -557,9 +571,37 @@
     nativeModule.cache();
     nativeModule.compile();
 
+    if (config.experimentalModules) {
+      const ns = NativeModule._nsCache[id] = Object.create(null);
+      if (typeof nativeModule.exports === 'function')
+        ns.default = nativeModule.exports;
+
+      const descriptors =
+        Object.getOwnPropertyDescriptors(nativeModule.exports);
+      for (const [name, d] of Object.entries(descriptors)) {
+        if (!d.enumerable || d.set !== undefined)
+          continue;
+        if (d.get !== undefined && !lazyDep().has(d.get))
+          ns[name] = (...args) => nativeModule.exports[name](...args);
+        else
+          ns[name] = nativeModule.exports[name];
+      }
+
+      if (id === 'events')
+        delete ns.EventEmitter;
```

Comment:
I think it would be better to remove this exceptions for now to maintain identity?